source/fundamentals/collations.txt

```diff
@@ -397,29 +385,24 @@ In this example, we demonstrate the following:
    The following code example uses imports from the
    ``import com.mongodb.client.model`` package for convenience.
 
-.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
-   :language: java
-   :dedent:
-   :start-after: start findOneAndUpdate
-   :end-before: end findOneAndUpdate
-
 Since "Günter" is lexically before "Gunter" using the
-``de@collation=phonebook`` collation in ascending order, the preceding operation
-returns the following update document:
+``de@collation=phonebook`` collation in ascending order, the following operation
```

Comment:
Suggest putting the "de@collation=phonebook" on line 378 in monospace as well (to avoid the email link)